src/demos/calcite-checkbox.html

```diff
@@ -32,846 +33,750 @@
+          <span>wrapped in calcite-label span with for</span>
+          <calcite-checkbox
+            id="light-wrapped-in-calcite-label-span-with-for"
+            name="light-wrapped-in-calcite-label-span-with-for"
+            scale="l"
+          ></calcite-checkbox>
+        </calcite-label>
+      </div>
+    </div>
+
+    <!-- 
+      **************************************************
+      * SIBLING CALCITE-LABEL
+      **************************************************
+    -->
+    <div class="parent">
+      <div class="child right-aligned-text">sibling calcite-label</div>
+
+      <!-- small -->
+      <div class="child">
+        <calcite-label for="light-sibling-calcite-label"
+          >sibling calcite-label
+          <calcite-checkbox id="light-sibling-calcite-label" name="sibling-calcite-label" scale="s"></calcite-checkbox>
+        </calcite-label>
+      </div>
+
+      <!-- medium -->
+      <div class="child">
+        <calcite-label for="light-sibling-calcite-label"
+          >sibling calcite-label
+          <calcite-checkbox id="light-sibling-calcite-label" name="sibling-calcite-label" scale="m"></calcite-checkbox>
+        </calcite-label>
+      </div>
+
+      <!-- large -->
+      <div class="child">
+        <calcite-label for="light-sibling-calcite-label"
+          >sibling calcite-label
+          <calcite-checkbox id="light-sibling-calcite-label" name="sibling-calcite-label" scale="l"></calcite-checkbox>
+        </calcite-label>
+      </div>
+    </div>
+
+    <!-- 
+      **************************************************
+      * WRAPPED IN NATIVE LABEL
+      **************************************************
+    -->
+    <div class="parent">
+      <div class="child right-aligned-text">wrapped in native label</div>
+
+      <!-- small -->
+      <div class="child">
+        <label>
+          <calcite-checkbox name="wrapped-in-label" scale="s"></calcite-checkbox>
+          wrapped in native label
+        </label>
+      </div>
+
+      <!-- medium -->
+      <div class="child">
+        <label>
+          <calcite-checkbox name="wrapped-in-label" scale="m"></calcite-checkbox>
+          wrapped in native label
+        </label>
+      </div>
+
+      <!-- large -->
+      <div class="child">
+        <label>
+          <calcite-checkbox name="wrapped-in-label" scale="l"></calcite-checkbox>
+          wrapped in native label
+        </label>
+      </div>
+    </div>
+
+    <!-- 
+      **************************************************
+      * WRAPPED IN NATIVE LABEL WITH FOR
+      **************************************************
+    -->
+    <div class="parent">
+      <div class="child right-aligned-text">wrapped in native label with for</div>
+
+      <!-- small -->
+      <div class="child">
+        <label for="light-wrapped-in-label-with-for">
+          <calcite-checkbox
+            id="light-wrapped-in-label-with-for"
```

Comment:
Can you make the checkbox ids and (wrapping label's `for` attributes) unique to each checkbox?